node/src/components/contract_runtime/operations.rs

```diff
@@ -131,8 +137,13 @@ pub fn execute_finalized_block(
         )?;
     }
 
-    // WARNING: Do not change the order of `deploys` as it will result in a different root hash.
-    for deploy in executable_block.deploys {
+    // WARNING: Do not change the order of `transactions` as it will result in a different root
+    // hash.
+    for txn in executable_block.transactions {
+        let deploy = match txn {
+            Transaction::Deploy(deploy) => deploy,
+            Transaction::V1(_) => continue,
```

Comment:
Why are we not executing V1 transactions here?

Reply:
As per comment above.